### PR TITLE
FABCN-391 v2.1.0 Release

### PR DESCRIPTION
**Pull request by heatherlp**
- also fixed nodeenv images to use node 12
- Also cherry pick commit 290ecb9d800e7f412a5a65441a945f48da22b576 from master

Signed-off-by: heatherlp <heatherpollard0@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
+## v2.1.0
+Wed  1 Apr 2020 16:12:36 BST
+
+* [1917562](https://github.com/hyperledger/fabric-chaincode-node/commit/1917562) [FABCN-392](https://jira.hyperledger.org/browse/FABCN-392) Update to Node 12
```
Comment (mbwhite) [inline CHANGELOG.md:4]: Previously we've hand edited this file to remove some of the 'minutae'... eg, Re-enable tests... etc.. Just to be the major functional changes.

File: release_notes/v2.1.0.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+v2.1.0
+------
```
Comment (mbwhite) [inline release_notes/v2.1.0.txt:2]: maybe a short line about the release-2.0 to release-2.x ...?

File: release_notes/v2.0.0.txt [hunk 3]
```diff
@@ -50,4 +50,4 @@ none
 
 Change Log
 ----------
-https://github.com/hyperledger/fabric-chaincode-node/blob/release-2.0/CHANGELOG.md#v2.0.0
+https://github.com/hyperledger/fabric-chaincode-node/blob/release-2.x/CHANGELOG.md#v2.0.0
```
Comment (jt-nti) [inline release_notes/v2.0.0.txt:53]: Should this be...

`https://github.com/hyperledger/fabric-chaincode-node/blob/release-2.x/CHANGELOG.md#v2.1.0`

?

File: release_notes/v2.1.0.txt [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Release Notes
+-------------
+There are minimal changes between v2.1.0 and v2.0.0, please see the change log for a full list of updates.
```
Comment (jt-nti) [inline release_notes/v2.1.0.txt:6]: Maybe it's just me but `v2.1.0 and v2.0.0` feels like it's the wrong way round.

File: release_notes/v2.1.0.txt [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+The release-2.0 branch has been renamed to release-2.x; the v2.1.0 release supercedes v2.0.0.
+A release-2.lts branch will be created when the 2.x LTS release has been decided, please see the proposed Fabric LTS strategy:
```
Comment (jt-nti) [inline release_notes/v2.1.0.txt:9]: I'd be tempted to state that this is not an LTS release and be a bit more vague about the future. Perhaps linking to the current LTS release as well as the LTS RFC

File: release_notes/v2.0.0.txt (hunk 3 above)
Comment (heatherlp) [reply release_notes/v2.0.0.txt:53]: Not for the 2.0.0.txt release note :)

File: release_notes/v2.1.0.txt (hunk 5 above)
Comment (heatherlp) [reply release_notes/v2.1.0.txt:9]: Think Line 10 is the LTS RFC?

File: release_notes/v2.0.0.txt (hunk 3 above)
Comment (jt-nti) [reply release_notes/v2.0.0.txt:53]: Good point!

**Comment by btl5037** [comment]
You need to update the KEEPALIVETIMEOUT here: https://github.com/hyperledger/fabric-chaincode-node/blob/master/tools/toolchain/network/docker-compose/docker-compose-base.yaml#L112
I set it to 120 but, 60 or 30 would probably also work

**Pull request merged**